### PR TITLE
Integrate Hailo 8 FW

### PR DESCRIPTION
**Pull request by rucoder**

### CODE REVIEW

File: pkg/fw/Dockerfile
Comment (rene) [inline pkg/fw/Dockerfile:65]: Well, I think we must take this with precaution. Although it's a firmware blob, so we are not talking about source code to be included on EVE, this written permission should be compatible with Apache 2.0, which means that it should not forbid us in anyway from any of the permissions guaranteed by our license: rights to use the software as it is, rights to change it (EVE), rights to share it... since it's a firmware blob, perhaps the most important is the right to share it as we wish (within EVE, of course)... I think it's important to include a license file here, we do have proprietary firmware in this package (i.MX, NVIDIA), but they have specific (proprietary) licenses that allows to share, pack with EVE, etc, i.e., which doesn't break the terms of our license...

Comment (rucoder) [reply pkg/fw/Dockerfile:65]: @rene yes, sure. I've submitted this PR as a placeholder so I do not loose the code. still waiting for confirmation and proper license file.

Comment (temerkhanov) [reply pkg/fw/Dockerfile:65]: It might be feasible to download it during VM deployment.

Comment (rucoder) [reply pkg/fw/Dockerfile]: > It might be feasible to download it during VM deployment.
not really, the end user must accept EULA but we managed to get approval from Hailo

File: pkg/fw/Dockerfile
Comment (rene) [inline pkg/fw/Dockerfile:59]: The destination directory (along with its child dirs) will be created by ADD, so you don't need this RUN here...

Comment (rucoder) [reply pkg/fw/Dockerfile]: fixed.

**Review by rene — APPROVED** [review]
LGTM

**Pull request merged**
